src/Utils/Logger.ts

```diff
@@ -51,10 +51,11 @@ export class Logger {
     if (typeof errorOrMessage === 'string') {
       args.unshift(messageOrArg);
       message = errorOrMessage;
+      const formattedMessage = format(message, ...args);
 
-      captureMessage(message, {
+      captureMessage(formattedMessage, {
         ...captureContext,
-        fingerprint: [message],
+        fingerprint: [formattedMessage],
```

Comment:
To deduplicate messages, you just need to make sure the fingerprint is unique enough. You can try to incorporate partial fingerprints and variables to achieve this:

```suggestion
        fingerprint: ['{{ transaction }}', '{{ message }}', ...args],
```

Reply:
Well, actually this line redundant now. This logic was used to group messages like `Critical error: %s` into single issue before this PR. But now having default fingerprint behaviour would be enough